python/pyarrow/_cuda.pyx

```diff
@@ -26,15 +26,17 @@ cdef class Context:
     """ CUDA driver context.
     """
 
-    def __cinit__(self, int device_number=0):
+    def __cinit__(self, int device_number=0, uintptr_t handle=0):
```

Comment:
Is this the recommended C type for this argument?

Reply:
I think yes because the `CUcontext` value (that the `handle` argument is holding) is actually a pointer to CUDA internal structure. `uintptr_t` should be an appropriate type for pointer values.